src/crimson/os/seastore/btree/btree_types.h

```diff
@@ -279,16 +280,20 @@ struct LBACursor : BtreeCursor<laddr_t, lba::lba_map_val_t> {
     assert(!is_indirect());
     return val->refcount;
   }
-  std::unique_ptr<LBACursor> duplicate() const {
-    return std::make_unique<LBACursor>(*this);
+  bool contains(laddr_t laddr) const {
+    return get_laddr() <= laddr && get_laddr() + get_length() > laddr;
+  }
+  boost::intrusive_ptr<LBACursor> duplicate() const {
```

Comment:
As discussed before, since `LBACursor` is changed to a `boost::intrusive_ptr` in this PR, the ptr itself can be copied, seems to me there is no need to copy-construct the `LBACursor` any more, right?